.gitignore

```diff
@@ -1,3 +1,6 @@
 _site
 .sass-cache
 .jekyll-metadata
+
+# OSX
+.DS_Store
```

Comment:
I don't know if I want this to be kept here—I think people should ignore these themselves with their git client.

Reply:
If you feel strongly, I can get rid of it and just request changes if it ever becomes an issue.